include/ttmlir/Dialect/TTNN/IR/TTNNOps.td

```diff
@@ -296,6 +296,20 @@ def TTNN_ReluOp : TTNN_ElementwiseUnaryOp<"relu",
     }];
 }
 
+def TTNN_RoundOp : TTNN_Op<"round"> {
+    let summary = "Eltwise round operation.";
+    let description = [{
+      Eltwise round with decimal option to choose between Banker's rounding or normal rounding.
+    }];
+
+    let arguments = (ins
+        Variadic<AnyRankedTensor>:$inputs,
```

Comment:
Why `Variadic` when `ttnn.round` takes just one tensor? Also it takes `std::optional<MemoryConfig>`.